api/comments/serializers.py

```diff
@@ -26,17 +28,19 @@ class CommentSerializer(JSONAPISerializer):
         'deleted',
         'date_created',
         'date_modified',
+        'page',
         'target'
     ])
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
     content = AuthorizedCharField(source='get_content', required=True, max_length=osf_settings.COMMENT_MAXLENGTH)
+    page = ser.CharField(read_only=True)
```

Comment:
Isn't the `ChoiceField` only for writable values? I made `page` read-only so that whenever a comment is created the page is determined from the root target.